components/viewer/modal/BackpackModal.vue

```diff
@@ -114,6 +135,85 @@ const objectMode = computed(() => {
   if (lockBackpackObjects.value) return ViewContext.BackpackDisabled;
   else return ViewContext.BackpackEnabled;
 });
+
+const backpackRef = ref<HTMLDivElement>();
+const isLoading = ref(false);
+const backpackToImage = async () => {
+  if (backpackRef.value && packRows.value.length >= 1) {
+    isLoading.value = true;
+    const $toast = useToast();
+    const toastGenerateImage = $toast.info('Generating image...', {
+      timeout: false,
+    });
+    // A hack, the DOM won't update until after the image is generated otherwise
+    const pause = new Promise((resolve) => setTimeout(resolve, 200));
```

Comment:
There's a `sleep` function in `utils.ts`

iirc, `await nextTick()` should work here.

Reply:
nextTick works, the sleep was leftover from my attempt with modern-screenshot.